### DISCUSSION TITLE
Different page_separator_width  between main window and Overview window

### DISCUSSION THREAD

**Discussion by qingshuizheng** [ahrm/sioyek#625] [Q&A] [answered]
Hi @ahrm ,
Thanks for fixing the question just now: https://github.com/ahrm/sioyek/discussions/623
There is a similar issue as the previous one, but regarding the different appearance of page_separator_width between the main window and overview window.

![Kapture 2023-02-06 at 06 39 30](https://user-images.githubusercontent.com/78839107/216965028-a2f29b24-cf75-4b3e-8a29-ef47cf187335.gif)

I record this gif (same could be found in the previous discussion gif, but it was scrolling too fast so i didnt notice it):

1. first part: set `page_separator_width 2`
2. view the main window page_separator_width appearance: normal 2 pixel -ish 
3. view the Overview window page separator appearance: huge gap (100 pixel -ish?) between pages

My prefs_user.config are attached if you need to check it (github does not support .config type file, so i change the extension to .txt): 
[prefs_user.txt](https://github.com/ahrm/sioyek/files/10662608/prefs_user.txt)

Actually i remove everything in the prefs_user.config, no improvements before-and-after.

Env:
MacOS Big Sur (11.6)
Sioyek version: Release v0.31.427, from here: https://github.com/hexomancer/sioyek/releases/download/v0.31.427/sioyek-release-mac.zip

Thanks!
Zheng

**Comment by ahrm** [comment, marked as answer]
This has nothing to do with page separator (we don't have page separators in overview window). What happened is we only render one page at a time in overview window (shouldn't be a problem since it is a pretty small window anyway) so what you are interpreting as page separator is the empty part after the top/bottom of the page. The problem is that instead of using the background color, we always used white as the  background of overview window which can be jarring. It should be fixed in 77e2bfba15893fd4a429a83679f9dc967415cd24, now we use the custom background color as the background.

**Reply by qingshuizheng** [reply]
Yeah that would be nicer looking, thanks for the improvement, I will try it out after the release is finished.

Another itchiness is, being over jumpy when scroll between pages within Overview:
![Kapture 2023-02-06 at 07 20 14](https://user-images.githubusercontent.com/78839107/216970433-c90a86b3-2445-4a56-adf3-2eef4c99264e.gif)

Are we able to fix that?

**Reply by ahrm** [reply]
I think this is fixed in the same commit (I think this is an illusion because we simultaneously change pages and scroll).

**Reply by qingshuizheng** [reply]
Tested. Perfect, thanks ahrm!
